services/navigation/client.go

```diff
@@ -186,6 +186,15 @@ func (c *client) GetObstacles(ctx context.Context, extra map[string]interface{})
 	return geos, nil
 }
 
+func (c *client) Paths(ctx context.Context, extra map[string]interface{}) ([]*Path, error) {
+	req := &pb.GetPathsRequest{Name: c.name}
```

Comment:
why are we not also including the `extra` fields?

Reply:
good catch! Forgot to include them on accident